### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
added products page

### CODE REVIEW

File: src/components/NavBar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+.Nav-header {
+  display: flex;
+  align-items: center;
+  background-color: var(--bg-lightWhite);
+  height: 12vh;
+  padding: 2.4rem 4.8rem;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  /* box-shadow: 0 4px 12px 0 rgba(0 , 0 ,0 , .8); */
```
Comment (Shah-Arsalan) [inline src/components/NavBar/Navbar.css:12]: romove this please

File: src/components/NavBar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import "./Navbar.css";
+import React from "react";
+import FavoriteBorderOutlinedIcon from "@mui/icons-material/FavoriteBorderOutlined";
```
Comment (Shah-Arsalan) [inline src/components/NavBar/Navbar.jsx:3]: Oh great to see material icons !!

**Review by Shah-Arsalan — APPROVED** [review]
code looks good to me . You can go ahead and merge

**Review by lomesshh — APPROVED** [review]
clean code
looks good to merge

**Pull request merged**